packages/utils/src/asciiUtils.ts

```diff
@@ -1,4 +1,5 @@
 // Regular expression to match non-ASCII characters
+// eslint-disable-next-line no-control-regex
 const nonAsciiPattern = /[^\x00-\x7F]/g;
```

Comment:
Let's avoid the control characters instead:

```suggestion
const nonAsciiPattern = /[^\x20-\x7E]/g;
```